bonito/models/configs/dna_r9.4.1@v3.1.toml

```diff
@@ -30,6 +30,8 @@ num_attn_heads = 1
 use_isab_attn = true
 isab_num_latents = 6
 
+weight_tie_attn_blocks = false
```

Comment:
for parameter saving when using ISAB blocks, which has twice the number of attention parameters than S(M)HA blocks